msteams-platform/apps-in-teams-meetings/enable-and-configure-your-app-for-teams-meetings.md

```diff
@@ -114,13 +114,9 @@ Messaging extension works as expected when a user is in an in-meeting view. The
 
 #### In-meeting dialog box
 
-The in-meeting dialog box is used to engage participants during the meeting and collect information or feedback during the meeting. Use the [SendNotificationSignal API](API-references.md#send-notification-signal-api) to trigger a bubble notification. As part of the notification request payload, include the URL where the content to be shown is hosted.
+The in-meeting dialog box is used to engage participants during the meeting and collect information or feedback during the meeting. Use the [NotificationSignal API](API-references.md#send-notification-signal-api) to trigger an in-meeting dialog box. As part of the notification request payload, include the URL where the content to be shown is hosted.
 
-In-meeting dialog must not use task module. Task module isn't invoked in a meeting chat. An external resource URL is used to display the content bubble in a meeting. You can use the `submitTask` method to submit data in a meeting chat.
-
-> [!NOTE]
-> * You must invoke the [submitTask()](../task-modules-and-cards/task-modules/task-modules-bots.md#submit-the-result-of-a-task-module) function to dismiss automatically after a user takes an action in the web view. This is a requirement for app submission. For more information, see [Teams SDK task module](/javascript/api/@microsoft/teams-js/microsoftteams.tasks?view=msteams-client-js-latest#submittask-string---object--string---string---&preserve-view=true). 
-> * If you want your app to support anonymous users, initial invoke request payload must rely on `from.id` request metadata in `from` object, not `from.aadObjectId` request metadata. `from.id` is the user ID and `from.aadObjectId` is the Microsoft Azure Active Directory (Azure AD) ID of the user. For more information, see [using task modules in tabs](../task-modules-and-cards/task-modules/task-modules-tabs.md) and [create and send the task module](../messaging-extensions/how-to/action-commands/create-task-module.md?tabs=dotnet#the-initial-invoke-request).
+In-meeting dialog must not use task module. Task module isn't invoked in a meeting chat. An external resource URL is used to display in-meeting dialog box. You can use the `submitTask` method to submit data in a meeting chat.
```

Comment:
After this paragraph, can you please insert a screenshot of in-meeting dialog? Similar to the meeting stage section? The screenshot in the "Desktop" section of this document would work, for example: https://docs.microsoft.com/en-us/microsoftteams/platform/apps-in-teams-meetings/design/designing-apps-in-meetings#desktop-2